src/test/java/com/academia/library/util/TestDataEntity.java

```diff
@@ -0,0 +1,29 @@
+package com.academia.library.util;
+
+import com.academia.library.dto.AuthorDto;
+import com.academia.library.dto.BookDto;
+import com.academia.library.dto.TagDto;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.HashSet;
+
+public class TestDataEntity {
```

Comment:
This name isn't brilliant. Please rename to TestDataCreator or something else

Reply:
Ok, thank